packages/paste-website/src/pages/primitives/listbox-primitive/index.mdx

```diff
@@ -0,0 +1,313 @@
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/primitives/listbox"
+  storybookUrl="/?path=/story/primitives-listbox--vertical-listbox"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+    Stack,
+  }}
+  noInline
+  showOverflow
+>
+  {defaultExample}
+</LivePreview>
+
+## Guidelines
+
+### About the Listbox Primitive
+
+The Listbox primitive is an unstyled, functional version of a listbox component. It can be used to build a component following the [WAI-ARIA Listbox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/).
+
+This unstyled primitive is to be used when the styled Listbox provided by Paste doesn&rsquo;t meet the requirements needed to solve a unique customer problem. At that point, you are welcome to fall back to this functional primitive to roll your own styled Listbox while still providing a functional and accessible experience to your customers.
+
+This primitive should be used to compose all custom Listboxes to ensure accessibility and upgrade paths.
+
+<Callout variant="warning" marginY="space70">
+  <CalloutHeading as="h3">Before you roll your own listboxes...</CalloutHeading>
+  <CalloutText>
+    We strongly suggest that all components built on top of this primitive get reviewed by the Design Systems team and
+    go through the UX Review process to ensure an excellent experience for our customers.
+  </CalloutText>
+</Callout>
+
+## Examples
+
+### Selection
+
+To make items inside the Listbox selectable, you can manage the selection state yourself and combine it with the state returned from the `useListboxPrimitiveState` hook. To do so, track which item is selected in a separate store of state. When rendering the Listbox, cross reference the rendered items with the selection state, and conditionally set `selected` on each `ListboxPrimitiveItem` that requires it.
+
+The `onSelect` event handler will fire whenever the item is clicked, or the spacebar or enter key is pressed. Use this to respond to your users' selection interactions.
+
+### Horizontal Listbox
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+  }}
+  noInline
+  showOverflow
+>
+  {horizontalExample}
+</LivePreview>
+
+### Listbox with groups
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveGroup,
+    ListboxPrimitiveItem,
+    Stack,
+  }}
+  noInline
+  showOverflow
+>
+  {groupsExample}
+</LivePreview>
+
+### Multiselect Listbox
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+  }}
+  noInline
+  showOverflow
+>
+  {multipleExample}
+</LivePreview>
+
+### Dual Listboxes
```

Comment:
Suggestion: can you add an explanation for why we have this example? Something like
> Listboxes can be used in any situation where a user needs to select items in a list or rearrange a list. One example is a UI where users need to move items between two lists like this: